ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/UserManagementServiceAnnotationsValidationTests.scala

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites.v1_8
+
+import java.nio.charset.StandardCharsets
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.v1.admin.user_management_service.{CreateUserRequest, CreateUserResponse}
+import com.daml.ledger.api.testtool.infrastructure.Assertions.{assertEquals, _}
+
+trait UserManagementServiceAnnotationsValidationTests { self: UserManagementServiceIT =>
+
+  userManagementTest(
+    "TestAnnotationsSizeLimits",
+    "Test annotations' size limit",
+    requiresUserExtensionsForHub = true,
+  )(implicit ec => { implicit ledger =>
+    val userId1 = ledger.nextUserId()
+    val largeString = "a" * 256 * 1024
```

Comment:
Same comment as for the parties: would it be possible to base the test on a constant?

Reply:
Ok, fixing the same as for parties